march_hardware_builder/src/robots/march4.yaml

```diff
@@ -21,21 +21,21 @@ march4:
       highVoltageNetOnOff: 3
       highVoltageNetEnableDisable: 2
   joints:
-    #    - right_hip_aa:
-    #        allowActuation: false
-    #        netNumber: 1
-    #        temperatureges: {slaveIndex: 8, byteOffset: 0}
-    #        imotioncube:
-    #          slaveIndex: 6
-    #          encoder: {resolution: 17, minPositionIU: 1, maxPositionIU: 3, zeroPositionIU: 2, safetyMarginRad: 0}
-    #
-    #    - left_hip_aa:
-    #        allowActuation: false
-    #        netNumber: 5
-    #        temperatureges: {slaveIndex: 8, byteOffset: 0}
-    #        imotioncube:
-    #          slaveIndex: 9
-    #          encoder: {resolution: 17, minPositionIU: 1, maxPositionIU: 3, zeroPositionIU: 2, safetyMarginRad: 0}
+    - right_hip_aa:
+        allowActuation: true
+        netNumber: 1
+        temperatureges: {slaveIndex: 8, byteOffset: 0}
+        imotioncube:
+          slaveIndex: 6
+          encoder: {resolution: 12, minPositionIU: 3688, maxPositionIU: 4070, zeroPositionIU: 3858, safetyMarginRad: 0.05}
+
+    - left_hip_aa:
+        allowActuation: true
+        netNumber: 5
+        temperatureges: {slaveIndex: 8, byteOffset: 0}
+        imotioncube:
+          slaveIndex: 9
+          encoder: {resolution: 12, minPositionIU: 1010, maxPositionIU: 1401, zeroPositionIU: 1194, safetyMarginRad: 0.05}
```

Comment:
The resolution of the `right_hip_aa` is 382 IU, while the `left_hip_aa` is 291 IU, is this acceptable?

Reply:
In the same vein is the zero position of `right_hip_aa` 170 IU from its start, and `left_hip_aa` 184 IU

Reply:
Could just be that the `minPositionIU` of the `right_hip_aa` is 10 to low, or the `left_hip_aa` too high.

Reply:
The min and max position of the joints was determined by pushing the joints to their physical min and max position. The difference in range is therefore caused by the hardware. This difference can therefore not be solved through a software solution.